app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -49,8 +43,8 @@ ns.SearchPage = do (ko
       $('#variablesModal').modal('show')
     $('.launch-customize-modal').click ->
       $('#customizeDataModal').modal('show')
-      
-    setTimeout initModal, 2000 if !window.edscportal
+    if (url.cleanPath()?.split('?')[0] in ["/search", "/", ""]) && current.preferences.serialize()?.doNotShowTourAgain == 'false'
+      $('#sitetourModal').modal('show')
```

Comment:
This new technique was not reading the preferences correctly somehow - the tour modal was coming up every time regardless of preference.